test/models/normalize_primo_test.rb

```diff
@@ -79,47 +79,47 @@ def missing_fields
     result = popcorn['results'].first
     assert_not_equal 'Rudolph, J.$$QRudolph, J.', result.authors.first.first
     assert_equal ['Rudolph, J.',
-                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Rudolph, J.&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
+                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Rudolph%2C%20J.&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
                  result.authors.first
   end
 
   test 'multiple authors in a single element are appropriately parsed' do
     result = monkeys['results'].second
     assert_not_equal ['Beran, Michael J ; Smith, J. David'], result.authors.first.first
     assert_equal ['Beran, Michael J',
-                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Beran, Michael J&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
+                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Beran%2C%20Michael%20J&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
                  result.authors.first
     assert_equal ['Smith, J. David',
-                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Smith, J. David&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
+                  'https://mit.primo.exlibrisgroup.com/discovery/search?query=creator,exact,Smith%2C%20J.%20David&tab=all&search_scope=all&vid=FAKE_PRIMO_VID'],
                  result.authors.second
```

Comment:
I just noticed that many things change automatically with my local styles check on save. Sorry for the noise. These are the only _intentional_ changes to this file